modules/sharedaddy/sharing-service.php

```diff
@@ -748,8 +748,17 @@ function sharing_display( $text = '', $echo = false ) {
 		$show = false;
 	}
 
-	// Allow to be used on P2 ajax requests for latest posts.
-	if ( defined( 'DOING_AJAX' ) && DOING_AJAX && isset( $_REQUEST['action'] ) && 'get_latest_posts' == $_REQUEST['action'] ) {
+	/**
+	 * Filter the default ajax action name
+	 *
+	 * @module sharedaddy
+	 *
+	 * @since 5.1.1
```

Comment:
Since it's a new filter, you can use the next version of Jetpack here:

```suggestion
	 * @since 7.2.0
```